packages/extension-ui/src/Popup/Signing/TransactionIndex.tsx

```diff
@@ -32,43 +28,44 @@ function TransactionIndex ({ className, index, onNextClick, onPreviousClick, tot
         <span className='totalSteps'>/{totalItems}</span>
       </div>
       <div>
-        <ArrowLeft
-          isActive={previousClickActive}
-          onClick={(): unknown => previousClickActive && onPreviousClick()}
+        <FontAwesomeIcon
```

Comment:
Nice! Also whacked some of the lef-over styled things I'm not a huge fan of.

Reply:
yes! trying to take care of those as I see them since I already have my eyes on the CSS. There will surely be some left after this PR, I'll take care of the rest in a dedicated PR I guess to keep this one under control (although I feel it's already too late).